application/views/admin/survey/activateSurvey_view.php

```diff
@@ -183,6 +187,15 @@ function alertPrivacy()
         <p class='col-sm-7 col-sm-offset-2'>
             <?php eT("Please note that once responses have collected with this survey and you want to add or remove groups/questions or change one of the settings above, you will need to deactivate this survey, which will move all data that has already been entered into a separate archived table."); ?><br /><br />
         </p>
+        <?php if($oSurvey->getIsExpiredByConfig()):?>
+        <div class="row">
+            <div class='col-sm-7 col-sm-offset-2'>
+
+                <div class="alert alert-danger"><?= Yii::t('app', 'NB! The survey you are activating is currently expired! The survey will not be available for participants.')?><div>
+            </div>
+
+        </div>
+        <?php endif;?>
```

Comment:
I also did it like that at first, but. It does not match the context well and it would give a wrong impression. By having a (past) datetime set for a survey does not actually mean it is expired. If the survey is not activated yet, then it is not expired by the definitions set here: 

https://github.com/LimeSurvey/LimeSurvey/blob/4dc6d4ff14cff15361eb81b1a07c9684905e453e/application/models/Survey.php#L1133..L1159

According to the `survey->state` the state is `inactive`. So having an `isExpired = true` while `->status = 'inactive'` is confusing.

Reply:
Ah, you mean isExpired must mean : `active== 'Y' && expires < "now"` ?

I really don't like this ByConfig … Maybe something like isDateExpired ?

Else : you put the comment where i just put idea about isStarted(ByConfig|ByDate) ;)